packages/mobile/src/components/toasts/Toast.tsx

```diff
@@ -74,9 +74,11 @@ export const Toast = (props: ToastProps) => {
       toValue: 1,
       useNativeDriver: true
     }).start(() => {
-      setTimeout(() => {
-        animateOut()
-      }, timeout)
+      if (timeout !== 'MANUAL') {
```

Comment:
so im coming around to some thoughts for the props/config of toast, so instead of overloading the "timeout" prop with "MANUAL", maybe we have a new prop/config value like noTimeout, or manualDismiss, or something along those lines? this is a smaller thing, but just feeling like from a user perspective, it's a bit weird to have a value "timeout" be a number or a string === "MAUNAL" know what i mean?

Reply:
Yea I can get behind this. I'll do some testing to see how this feels. Want to make sure that I maintain an easy method to have the toast dismiss itself based on changes in the slice